src/structures/Chat.js

```diff
@@ -270,6 +270,43 @@ class Chat extends Base {
     async changeLabels(labelIds) {
         return this.client.addOrRemoveLabels(labelIds, [this.id._serialized]);
     }
+
+    /**
+     * @param {number} days The value to set the message disappear after time to. Valid values are 0, 1, 2, 3. 0 for message expiration removal, 1 for 24 hours message expiration, 2 for 7 days message expiration, 3 for 90 days message expiration
+     * @returns {Promise<boolean>} Returns true if the operation completed successfully, false otherwise
+     */
+    async setDisappearing(days) {
+        switch (days) {
+            case 0:
+                days = 0;
+                break;
+            case 1:
+                days = 86400;
```

Comment:
I dont think this is 86400 days :-)
I think you should create variable that represent what the number is